.github/issue_template.md

```diff
@@ -0,0 +1,19 @@
+<!--
+Hi!  Thanks for taking the time to report feedback.
+
+Are you concerned with running your own instance of Cromwell,e.g. config issues?  Then please report your issue to the WDL Forum:
+https://gatkforums.broadinstitute.org/wdl/categories/ask-the-wdl-team
```

Comment:
Ideally we'd be able to say:
* WDL question -> WDL forum
* CWL question -> CWL gitter/forum
* Config question -> read the docs/Cromwell forum
* Firecloud question -> FireCloud forum
* Bug: fill in this template...